docs/setup/README.md

```diff
@@ -1,22 +1,33 @@
-# Minimal Setup
+# Quickstart guide
 
-This page contains the minimal steps to use the MKDocs Tech Docs Template. Some additional customisation is provided in [customise](customise.md). You are encouraged to read the Material for MkDocs [guidance](https://squidfunk.github.io/mkdocs-material/) for more detailed information on setting up and customisation.
+This page provides a guide on how to configure the MKDocs Tech Docs Template. In a few simple steps, you can begin building your documentation site.
 
-## Installation
+## Before you begin
 
-MkDocs Tech Docs template is published as a Python package and can be installed with pip, ideally by using a virtual environment. Open up a terminal and install with:
+In order to use the MKDocs Tech Docs Template, you require the following:
+
+- a device with permissions to install new packages
+- Python 3 and Pip installed on your device
+- GitHub account with [a configured SSH key][gh_ssh]
+- GitHub repository to store your documentation (see [GitHub's documentation][gh_newRepo] for instructions)
+
```

Comment:
You don't need a GitHub account or a GitHub repository, for example you could use BitBucket